tests/utils.py

```diff
@@ -0,0 +1,25 @@
+import asyncio
+
+
+@asyncio.coroutine
+def populate(es, index, doc_type, n, *, loop):
+    coros = []
+
+    yield from es.transport.perform_request('PUT', index)
```

Comment:
no. I need to create index itself without any documents.
It's for case when `n == 0` --> index is created but not populated.